frontend/src/components/regions/GroupCardSummary/CellContent/Presentation.js

```diff
@@ -115,9 +87,7 @@ export default ({ card, handleClickOpenCardDetails }) => {
               </TableCell>
               <TableCell>
                 <Paper className={classes[card.status]} variant="outlined">
```

Comment:
Are you sure this Paper shopuld still be here? It looks like it is a repeat of what is inside frontend/src/components/widgets/CardStatusChip.js and the classname doesn't make sense.

There are a lot of people on the team, it's best if everyone does their bit to keep things tidy.